native/packages/react-native-backpack/libraries/Panel/index.js

```diff
@@ -16,5 +16,30 @@
  * limitations under the License.
  */
 /* @flow */
+import React, { Component, type Node } from 'react';
+import PropTypes from 'prop-types';
+import { requireNativeComponent } from 'react-native';
 
-// Not much to see here
+type Props = {
+  children: Node,
+  padded: boolean,
```

Comment:
Is `padded` non-functional right now? I tried applying it in the Storybook and through Interface Builder and it doesn't seem to have an effect.

Reply:
Yeah pretty much, the React Native part of this implementation doesn't really work.